src/views/Vaults.tsx

```diff
@@ -1,16 +1,36 @@
 import { useAtomValue } from 'jotai';
 import { appAtom, leaderAtom, networkConfigAtom } from 'store/app';
-import { useEffect } from 'react';
+import { FunctionComponent, useEffect } from 'react';
 import { watchVaultFactory } from 'service/vaults';
+import AdjustVault from 'components/AdjustVault';
 import CreateVault from 'components/CreateVault';
 import ManageVaults from 'components/ManageVaults';
 import MainContentWrapper from 'components/MainContentWrapper';
-import { useVaultStore, viewModeAtom, ViewMode } from 'store/vaults';
+import {
+  useVaultStore,
+  viewModeAtom,
+  ViewMode,
+  vaultKeyToAdjustAtom,
+} from 'store/vaults';
+import ReactViewSlider from 'react-view-slider';
+import type { VaultKey } from 'store/vaults';
 
-type PathDescriptionProps = { mode: ViewMode };
+type PathDescriptionProps = { mode: ViewMode; adjustVaultKey: VaultKey | null };
 
-const PathDescription = ({ mode }: PathDescriptionProps) => {
-  if (mode === ViewMode.Create) {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const ViewSlider = ReactViewSlider as unknown as FunctionComponent<any>;
```

Comment:
This package doesn't export types so this is the only way I found to get ts to quiet down.

Reply:
Ah. Consider explaining that in a code comment.

I looked at the package and it had Flow types so we could convert to TS and publish a DefinitelyTyped definition but prolly not worth it.

Reply:
Added a comment